### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "connect ECONNRESET 20.205.243.168:443"

### DISCUSSION THREAD

**Discussion by JasonSloan** [community/community#64482] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

IDE : vscode
VPN : off
platform : ubuntu
I had installed copilot extensions in my vscode  before using ubuntu which worked well, however when i launched vscode as usual, a window appeared : "GitHub Copilot could not connect to server. Extension activation failed: "connect ECONNRESET 20.205.243.168:443".
I tried to test the copilot function in pycharm, it worked well.
I tried to restart the vscode, it remained as bad.
I tried to reinstall the extension, it remained as bad.
I tried to change another server, it remained as bad.
How can i fix it ?

**Comment by davecheney** [comment, marked as answer]
20.205.243.168 is not one of the IP addresses of our servers. Have you hard coded something in your /etc/hosts file?

**Reply by JasonSloan** [reply]
Thanks for your answer! No i didnt modify the hosts file and it looks like below:
127.0.0.1       localhost
::1     localhost ip6-localhost ip6-loopback
fe00::0 ip6-localnet
ff00::0 ip6-mcastprefix
ff02::1 ip6-allnodes
ff02::2 ip6-allrouters
172.17.0.2      autodl-container-e7da1195fa-fbf7d1d3

**Reply by davecheney** [reply]
20.205.243.168 is registered to Azure in Singapore. We don't have an instance there.

**Reply by davecheney** [reply]
I think that IP is one of the addresses of api.github.com. Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by JasonSloan** [reply]
Thank you very much! Thats exactly the reason! The host of the server somehow blocked the copilot in recent days.  I tested to connect to copilot in another server and it succeeded! Thumbs up for your advice!

**Comment by arafajs** [comment]
1. Make sure that your internet connection is working properly. Try opening a web browser and loading a website.
2. Check your firewall settings to make sure that GitHub Copilot is not being blocked.
3. Try restarting your computer.
4. Uninstall and reinstall the GitHub Copilot extension.
5. If you are still having problems, you can contact GitHub Copilot support for help.
